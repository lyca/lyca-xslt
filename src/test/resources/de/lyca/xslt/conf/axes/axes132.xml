<?xml version="1.0"?>
<root>
	<child number="1" />
	<child number="2" />
	<child number="3" />
	<child number="4" />
	<child number="5">
		<grand-child number="1" />
		<grand-child number="2" />
		<grand-child number="3" />
		<grand-child number="4" />
	</child>
	<child number="6" />
	<child number="7" />
	<child number="8" />
</root>
