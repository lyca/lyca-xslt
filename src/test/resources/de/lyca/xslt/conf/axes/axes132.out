<?xml version="1.0" encoding="UTF-8"?><out>
<child number="2"/>
<child number="5">
		<grand-child number="1"/>
		<grand-child number="2"/>
		<grand-child number="3"/>
		<grand-child number="4"/>
	</child>
<child number="7"/>
</out>
